go.mod

```diff
@@ -4,7 +4,7 @@ go 1.15
 
 require (
 	github.com/ONSdigital/dp-healthcheck v1.0.5
-	github.com/ONSdigital/dp-mongodb/v2 v2.3.0
+	github.com/ONSdigital/dp-mongodb/v2 v2.3.1
```

Comment:
would it be possible to use 2.4.0 here? (it has the logging fixes in)

Reply:
Resolved